WalletWasabi/Services/WalletService.cs

```diff
@@ -1266,6 +1266,9 @@ public Operation(Script script, Money amount, string label)
 				.SendFees(fee)
 				.BuildTransaction(sign);
 
+			tx.Inputs.Shuffle();
+			tx.Outputs.Shuffle();
```

Comment:
Are you sure NBitcoin is not already shuffling inputs and outputs?

Reply:
No, wait! it is shuffling the transaction even when it is already signed? That invalidates the transaction. In case NBitcoin is not already shuffling the inputs/outputs, you could do it only if the transaction is not signed.

Reply:
Looks like NBitcoin is actually already shuffling, but it the transaction isn't signed yet at this point